src/aosm/azext_aosm/inputs/arm_template_input.py

```diff
@@ -109,3 +108,5 @@ def _generate_schema_from_arm_params(
                     )
```

Comment:
Can we change the `key` and `value` in this for loop to be a bit more descriptive? I think something like `parameter_name` and `parameter_value` would be better. If you can make it even more descriptive, even better

Reply:
Done